source/accounts/vsc_user_groups.rst

```diff
@@ -124,5 +125,63 @@ Working with file and directory permissions
 
       ::
 
-         [user@foo]# find /path/to/directory -type d -exec chmod g+s '{}' \\;
+         find /path/to/directory -type d -exec chmod g+s '{}' \\;
+
+
+.. _sharing_via_vsc_groups:
+
+Sharing files and folders with other VSC users
+----------------------------------------------
+
+VSC users can share files/folders with other VSC users via `VSC groups <vsc_groups>`.
```

Comment:
I'm not sure this the correct approach:
* this can create an explosion of groups in the VSC account page
* the share depends on the permissions of the top level directory of `$VSC_DATA`, which for instance in our case it is controlled by us and we do reset those permissions on each sync operation with the VSC account page

I would say that Globus is the preferred way to share data between users that are not part of the same Project or VO.

@jooghe maybe we should discuss this in the CUE meeting